apps/api/src/scripts/send-data-extract.ts

```diff
@@ -94,7 +94,7 @@ export const flattenAndTransformFormData = (submissions: SubmissionEntity[]) =>
       nonClinicalJobTitle: credentialInformation?.nonClinicalJobTitle,
 
       deployAnywhere: booleanToYesNo(preferencesInformation?.deployAnywhere),
-      VancouverRegion: healthAuthorities.VancouverRegion?.lhas.join(', '),
+      VancouverCoastal: healthAuthorities.VancouverCoastal?.lhas.join(', '),
```

Comment:
@dbayly-freshworks this was just a bug, coastal wasn't populating in the extracts